src/Tizen.Applications.Common/Tizen.Applications/ApplicationEventArgs.cs

```diff
@@ -0,0 +1,32 @@
+﻿/*
+ * Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+using System;
+
+namespace Tizen.Applications
+{
+    /// <summary>
+    /// An extended EventArgs class, which contains event data.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class ApplicationEventArgs : EventArgs
+    {
+        /// <summary>
+        /// The message passed from the event system.
+        /// </summary>
+        /// <since_tizen> 3 </since_tizen>
+        public Bundle EventData { get; internal set; }
```

Comment:
Why not `Data`? It is already named as `EventArgs`
So, `Event` is duplication of word. 
```
eventArgs.EventData vs eventArgs.Data
```